icons/framework.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Framework</title><path d="M7.9671.4095c-.6012.1103-2.0945.902-3.0515 1.6135-.5913.446-.7416.5913-.887.887-.1152.2404-.1202.2855-.1352 1.5733-.005.7316-.035 1.4732-.0652 1.6536-.0902.5512-.3908 1.1124-.8167 1.5182-.2305.2155-.3458.2907-1.5283.9872-.957.5612-1.1124.6965-1.2878 1.0723-.1653.3658-.1954.7265-.1954 2.31 0 1.4881.025 1.8038.1654 2.2047.1252.3658.3858.5913 1.2877 1.1224.441.2556.937.5562 1.1024.6614.6163.3959.962.8468 1.2276 1.5985.0401.1052.0702.5512.0802 1.1524.0401 2.0194.0451 2.1196.1453 2.305.1604.3057.2806.441.6715.7416 1.1725.907 2.756 1.7387 3.3873 1.7838.3758.0251.5812-.06 1.5333-.6263.907-.5412 1.5383-.8769 1.6535-.8819.0401-.005.1754-.025.3057-.0551.3507-.0702.8719-.025 1.2577.1152.1904.0652.8218.4059 1.408.7466 1.4331.8418 1.5634.8669 2.4753.451.2055-.0952.3759-.1704.3859-.1704.02 0 1.0071-.5511 1.2176-.6764.2255-.1353.7366-.5011 1.0673-.7667.2555-.2004.4159-.3758.5211-.5662l.1503-.2706.0301-1.5533c.025-1.403.04-1.5784.1353-1.829.2906-.7816.6865-1.2276 1.4782-1.6836 1.2326-.7015 1.7337-1.0272 1.884-1.2176.1754-.2205.3007-.5311.3358-.8318.01-.1102.04-.3708.0651-.5762.0451-.3909.035-2.2148-.02-2.6056-.0602-.486-.0802-.5813-.1353-.7517-.0802-.2605-.3057-.5662-.5262-.7115-.1052-.0701-.6313-.3908-1.1675-.7065-.5361-.3207-1.0472-.6414-1.1274-.7216-.2556-.2405-.5061-.5762-.6414-.8668-.2405-.5262-.2806-.8168-.2806-2.275-.005-1.3127-.005-1.3528-.1253-1.6084-.1202-.2656-.3257-.501-.6213-.7265-.0952-.0702-.2305-.1704-.3006-.2305-.2105-.1604-.9521-.6414-1.2878-.8218C16.4855.4847 15.814.2892 15.353.4747c-.0952.04-.506.2706-.922.5211-1.5833.942-1.7036.9921-2.4051.9971-.7667.005-.907-.05-2.5104-1.0071C8.6235.4546 8.3279.3444 7.967.4095zm4.9307 4.2492c.7466.0952 1.5433.3407 2.2097.6764.3408.1704.907.5011.952.5512.0151.015.1053.0852.2005.1503.2606.1854.3758.2857.8318.7416.7516.7567 1.408 1.824 1.7287 2.8161.1654.5111.1604.496.2806 1.0523.0902.4309.1303 1.894.0652 2.2799-.0251.1353-.0602.3407-.0752.451-.0802.511-.3207 1.2526-.5863 1.8239-1.0572 2.2548-3.1768 3.8282-5.577 4.149-.5662.075-2.1296.035-2.2799-.0602-.02-.01-.1102-.0351-.2054-.0501-.7516-.1303-2.0244-.7015-2.6607-1.1976-.0702-.0601-.2055-.1604-.2957-.2305-.5762-.426-1.4581-1.4982-1.859-2.2498-.3908-.7416-.7215-1.7689-.8268-2.5856-.05-.3808-.05-1.5584 0-1.9542.0752-.5863.2907-1.373.5312-1.9542.5812-1.393 1.7387-2.7209 3.0465-3.4875.7917-.461 1.9342-.8469 2.751-.922.1904-.02.3507-.0401.3607-.0451.025-.03 1.1475.01 1.408.045z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Framework</title><path d="M23.186 9.07 21.41 8.019a2.78 2.78 0 0 1-1.344-2.391V3.523c0-.431-.19-.837-.516-1.108A11.965 11.965 0 0 0 16.317.493a1.356 1.356 0 0 0-1.193.091L13.347 1.64a2.622 2.622 0 0 1-2.688 0L8.882.584a1.348 1.348 0 0 0-1.194-.09 11.93 11.93 0 0 0-3.231 1.918 1.44 1.44 0 0 0-.516 1.108v2.104c0 .986-.51 1.897-1.344 2.392L.823 9.068c-.363.215-.61.588-.675 1.013A12.24 12.24 0 0 0 0 12.001c0 .651.048 1.292.145 1.916.065.425.312.801.675 1.016l1.774 1.052a2.78 2.78 0 0 1 1.344 2.392v2.104c0 .431.191.837.516 1.108.965.8 2.054 1.452 3.231 1.919.393.155.831.124 1.194-.091l1.777-1.055a2.622 2.622 0 0 1 2.688 0l1.777 1.055c.363.215.804.246 1.193.091a11.973 11.973 0 0 0 3.232-1.92 1.44 1.44 0 0 0 .516-1.107v-2.104a2.78 2.78 0 0 1 1.344-2.392l1.774-1.052c.363-.215.61-.588.675-1.016.094-.624.145-1.265.145-1.916 0-.652-.048-1.293-.145-1.917a1.41 1.41 0 0 0-.67-1.013zM12.003 19.41c-3.981 0-7.21-3.317-7.21-7.407s3.229-7.406 7.21-7.406c3.98 0 7.21 3.316 7.21 7.406s-3.23 7.407-7.21 7.407z"/></svg>
```
Unfortunately I still see issues. To avoid dragging this further, I have suggested this change.
I will request @adamrusted to review the PR